migrations/sql/V1.35__alter_mine_expected_document.sql

```diff
@@ -0,0 +1,12 @@
+UPDATE mine_expected_document set exp_document_status_guid =
+(
+    SELECT exp_document_status_guid 
+    FROM mine_expected_document_status 
+    WHERE description = 'Not Received'
+    limit 1
+) 
+    WHERE exp_document_status_guid IS NULL;
+
+COMMIT;
+
+ALTER TABLE mine_expected_document ALTER COLUMN exp_document_status_guid SET NOT NULL;
```

Comment:
The model should be updated to reflect the non-nullable column

Reply:
I missed that, good chatch!